src/FastField.tsx

```diff
@@ -36,12 +36,27 @@ export type FastFieldAttributes<T> = GenericFieldHTMLAttributes &
  * Custom Field component for quickly hooking into Formik
  * context and wiring up forms.
  */
-export class FastField<Values = {}, Props = {}> extends React.Component<
-  FastFieldAttributes<Props>,
+export function FastField<Values = {}, Props = {}>(
```

Comment:
Can you just use connect()?

Reply:
I tried, but typescript and HOC's don't play well, can spend time trying to make it work if you insist.

Reply:
Just tried looking at it again. I can't figure out a way to use connect() and still preserve the generic arguments.

Reply:
I changed it to use connect() the same way as prior to 2.0, but this approach kind of makes it pointless for FastFieldInner to have generic arguments in the first place.

Your call.